writing-tests/remove-vowels/remove-vowels-in-array.test.js

```diff
@@ -1,11 +1,7 @@
 let removeVowelsFromWords = require("./remove-vowels-in-array");
```

Comment:
Code looks good 👍

On minor thing to note, ideally you would want the test and code files to have similar name.

You code file is named remove-vowels.js, generally your unit test files should have the same name too, so -- remove-vowels.test.js